packages/@aws-cdk/aws-ecs/lib/base/from-service-attributes.ts

```diff
@@ -47,7 +47,9 @@ export function fromServiceAtrributes(scope: Construct, id: string, attrs: Servi
     });
   } else {
     arn = attrs.serviceArn as string;
-    name = stack.splitArn(arn, ArnFormat.SLASH_RESOURCE_NAME).resourceName as string;
+    const resourceName = stack.splitArn(arn, ArnFormat.SLASH_RESOURCE_NAME).resourceName as string;
+    const resourceNameSplit = resourceName.split('/');
+    name = resourceNameSplit.length === 1 ? resourceName : resourceNameSplit[1];
```

Comment:
This part is overlapping with the changes that @jumic  is making in this PR https://github.com/aws/aws-cdk/pull/18140/files. 

Let's keep this PR to only changing `serviceArn` public property for Ec2 and Fargate Service that are created with the BaseService constructor. And we can leave the necessary changes for the `serviceArn` and `serviceName` public props for fromXXX() services to @jumic 's PR.

Reply:
Same comment for the changes in `ec2-service.ts` and `fargate-service.ts`